.github/workflows/conformance-gke.yaml

```diff
@@ -383,7 +397,7 @@ jobs:
           target_url: ${{ env.check_url }}
 
       - name: Set commit status to cancelled
-        if: ${{ cancelled() }}
+        if: ${{ !env.ACT && cancelled() }}
```

Comment:
I feel like if we add new jobs here, it's easy for someone to forget to add this to the conditions, but not sure we can do anything about that. :shrug: